Makefile

```diff
@@ -19,6 +19,7 @@ OSM_SRID?=3857
 OSM_UNITS?=meters
 OSM_DB_CONNECTION?=host=localhost dbname=osm user=osm password=osm port=5432
 OSM_EXTENT?=-20000000 -20000000 20000000 20000000
+OSM_FORCE_POSTGIS_EXTENT?=0
```

Comment:
why OSM_ prefix? Why postgis? FORCE_EXTENT looks better to be.

Reply:
OSM_ prefix because all other have same prefix.
POSTGIS because I only apply extent on postgis layers (no performance problem with other layers, and land_polygons are available for the whole world).